jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
+REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" "image-service")
```

Comment:
Add ```image-service``` directly to ```REPOS``` is easier but it will lead the pr gate clone this repo every time even it's not a images service pr gate . IMO it will be better if a var is defined to determine if this is an image service test.
Then this extra image-service repo clone can be ignored when uncessary.

The var can be defined in ```if(repos.contains("image-service")){```